pkg/policy/api/rule.go

```diff
@@ -235,6 +248,10 @@ type EgressRule struct {
 	//
 	// +optional
 	ToEntities []Entity `json:"toEntities,omitempty"`
+
+	// ToServices is a list of services to which the endpoint subject
+	// to the rule is allowed to initiate connections.
```

Comment:
Needs example and if it's an optional field or not.

Reply:
Done